eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -142,7 +148,10 @@ public function load(array $configs, ContainerBuilder $container)
      */
     public function getConfiguration(array $config, ContainerBuilder $container)
     {
-        return new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration = new Configuration($this->getMainConfigParser(), $this->suggestionCollector);
+        $configuration->setSiteAccessConfigurationFilters($this->siteaccessConfigurationFilters);
```

Comment:
That is what I did guess, indeed. In that particular case, since we iterate over the property, and it is initialized as an array, I think we're good, and I'd rather leave it as it is.